### PR TITLE
Fix votour after the change of representation of opaques.

### LINKED ISSUE #10487: `votour` should be tested
It seems that we don't test `votour`. Since it relies on a lot of unsafe features (dynamic casts), it probably means that we keep breaking it in practice...

### LINKED ISSUE #10856: votour seems broken by changes in vo file layout
It seems `votour` is broken:
```
$ checker/votour.exe _build/default/theories/Init/Notations.vo
Welcome to votour !
Enjoy your guided tour of a Coq .vo or .vi file
Object sizes are in words (64 bits)
At prompt, <n> enters the <n>-th child, u goes up 1 level, x exits

File format: 8991
Fatal error: exception End_of_file
```
`git bisect` blames 1cdfb1f9270e399a784b346c3f8d6abbc4477552:
```
commit 1cdfb1f9270e399a784b346c3f8d6abbc4477552
Author: Pierre-Marie Pédrot <pierre-marie.pedrot@inria.fr>
Date:   Tue Jun 4 10:35:14 2019 +0200

    Remove the discharge segment from vo files.
    
    Since the introduction of delayed section substitution, the opaque table
    was already containing the same information.
```

### PR DESCRIPTION
**Pull request by ppedrot**
Fixes #10856.
Fixes #10487

### CODE REVIEW

**Comment by ppedrot** [comment]
@maximedenes I added a commit to test votour, fixing #10487. Hopefully my makefile-fu was enough to sort out this.

**Comment by maximedenes** [comment]
@ppedrot Thanks a lot!

**Comment by ppedrot** [comment]
Test-suite broken, as expected. Let's investigate.

**Review by SkySkimmer — COMMENTED** [review]
Why use votour.byte instead of votour?

**Comment by ppedrot** [comment]
The misc tests are using the byte programs for some reason.

**Comment by SkySkimmer** [comment]
I don't know which one you looked at, for instance universes.sh uses native $coqc

Also does make based build install votour?

**Comment by ppedrot** [comment]
I reused the TOOLS target, let's see if this solves the test-suite.

File: test-suite/misc/votour.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+command -v "${BIN}votour" || { echo "Missing votour"; exit 1; }
+
+"${BIN}votour" ../theories/Init/Logic.vo < /dev/null
```
Comment (SkySkimmer) [inline test-suite/misc/votour.sh]: should be something stupid like `$($coqc -where)/theories/Init/Logic.vo`
Perhaps easier would be to compile a dummy file and run votour on that vo, that way you control where the vo is.

**Comment by gares** [comment]
I suggest you votout a file part of the test suite, there are some in prerequisites/.

**Comment by ppedrot** [comment]
Can we at least merge the first commit? Breaking votour is uncommon, as it reuses the same data representation as the checker. It requires changing the layout of the vo segments, which doesn't happen every morning.

**Comment by maximedenes** [comment]
@gares' suggestion seems like 2 minutes of work. I'll do it.

**Comment by maximedenes** [comment]
@ppedrot this PR seems to reveal a serious build problem of votour on macOS.

**Comment by ppedrot** [comment]
Reading the Makefile, it seems you're missing a entry in the `distributed_exec` list from configure.ml. I'll push a patch.

**Comment by ppedrot** [comment]
CI green indeed, this is ready to go. Who is assigning?

**Comment by ppedrot** [comment]
ping

**Comment by maximedenes** [comment]
I'll take it.

**Review by maximedenes — APPROVED** [review]

**Comment by maximedenes** [comment]
Maybe a quick approval from @SkySkimmer or @ejgallego would be nice.

**Review by SkySkimmer — APPROVED** [review]
Seems fine

**Pull request merged**